integration_test/src/main.rs

```diff
@@ -914,20 +919,23 @@ fn test_create_wallet(cl: &Client) {
             blank: None,
             passphrase: None,
             avoid_reuse: None,
+            descriptors: None,
```

Comment:
Bitcoin Core 23 now creates descriptor wallets by default. So in order for these tests to pass, this needs to be `descriptors: Some(false)` for each of these `WalletParams`.